docs/resources/ekscluster.md

```diff
@@ -123,7 +123,7 @@ resource "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
         }
 
         update_config {
-          max_unavailable_nodes = "10"
+          max_unavailable_nodes = "4"
```

Comment:
Is 4 the limit?

Reply:
Since this is docs and example, I will be resolving this comment. 
@DheerajSShetty ,can you address this in the comment or offline.

Reply:
No, it's just a working example. It can be anything between min and max.